code/modules/religion/rites.dm

```diff
@@ -134,10 +134,66 @@
 					/obj/item/organ/cyberimp/eyes/hud/medical,
 					/obj/item/organ/cyberimp/mouth/breathing_tube,
 					/obj/item/organ/cyberimp/chest/thrusters,
+					/obj/item/organ/cyberimp/chest/nutriment,
+					/obj/item/organ/cyberimp/arm/toolset,
+					/obj/item/organ/wings/cybernetic,
 					/obj/item/organ/eyes/robotic/glow)
 	new blessing(altar_turf)
 	return TRUE
 
+
+/datum/religion_rites/machine_implantation
+	name = "Machine Implantation"
+	desc = "Apply a provided upgrade to your body. Place a cybernetic item on the altar, then buckle a someone to implant them, otherwise it will implant you."
```

Comment:
```suggestion
	desc = "Apply a provided upgrade to your body. Place a cybernetic item on the altar, then buckle someone to implant them, otherwise it will implant you."
```